includes/Ajax_Handlers/WC_Stripe_Checkout_Sessions_Ajax_Handler.php

```
@@ -115,20 +113,20 @@ public function create_checkout_session(): void {
 				],
 			];
 
-			$checkout_session = WC_Stripe_API::request( $request, 'checkout/sessions' );
+			$checkout_session = \WC_Stripe_API::request( $request, 'checkout/sessions' );
 
 			if ( ! empty( $checkout_session->error ) ) {
 				$message = empty( $checkout_session->error->message ) ? __( 'Checkout Sessions API returned an error', 'woocommerce-gateway-stripe' ) : $checkout_session->error->message;
-				throw new Exception( $message );
+				throw new \Exception( $message );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '120', column '15'). (undefined)

(MissingImport)